package.json

```diff
@@ -45,7 +45,8 @@
     "widget"
   ],
   "dependencies": {
-    "react": "0.13.x",
+    "react": "0.14.x",
+    "react-test-utils": "0.0.1",
```

Comment:
This (react-test-utils) should be a `devDependency`, right? Because the end-user doesn't need it?